helper.psm1

```diff
@@ -17,7 +17,22 @@ $env:TMP = "$INNERWORKDIR\tmp"
 $env:CLCACHE_DIR="$INNERWORKDIR\.clcache.windows"
 
 $global:GENERATOR = "Visual Studio 15 2017 Win64"
-Import-Module VSSetup -ErrorAction Stop
+
+$global:launcheableTests = @()
+$global:maxTestCount = 0
+$global:testCount = 0
+$global:portBase = 10000
+
+$global:ok = $true
+
+################################################################################
+# Utilities
+################################################################################
+
+#ToDo
+#Function transformBundleSniplet
+#{   
+#}
```

Comment:
Why is this needed?

Reply:
It is just a reminder to implement this function